vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -33,50 +33,70 @@
             <ul
-              class="tw-text-[1rem] tw-leading-[1.125rem] md:tw-text-[1.09375rem] md:tw-leading-[1.28125rem] xl:tw-text-[1.1875rem] xl:tw-leading-[1.4375rem] tw-hidden lg:tw-flex lg:tw-flex-row tw-flex-wrap lg:tw-items-center tw-justify-evenly xl:tw-justify-end tw-w-full"
+              class="tw-hidden lg:tw-flex lg:tw-flex-row tw-flex-wrap lg:tw-items-center tw-justify-evenly xl:tw-justify-end tw-w-full tw-text-[1rem] md:tw-text-[1.09375rem] xl:tw-text-[1.1875rem] tw-leading-[1.125rem] md:tw-leading-[1.28125rem] xl:tw-leading-[1.4375rem]"
+              @mouseleave="!showContributionMenu"
             >
               <li
                 v-for="navbar in navbars"
                 :key="navbar"
                 class="tw-my-2 tw-py-2"
               >
                 <router-link
+                  @mouseover="
+                    showContributionMenu = navbar.name == 'Contribution'
+                  "
+                  @mouseleave="!showContributionMenu"
                   v-if="!navbar.target"
                   :to="navbar.url"
                   @click.native="mixpanel.track(navbar.event)"
-                  class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] lg:tw-mr-[8px] xl:tw-mr-[30px]"
+                  class="tw-inline-block tw-relative tw-mr-[20px] sm:tw-mr-[30px] lg:tw-mr-[8px] xl:tw-mr-[30px] tw-group"
                   :class="[
                     navbar.className
                       ? navbar.className
                       : navbar.class +
-                        ' after:tw-absolute after:tw-w-full after:tw-h-[3px] after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent',
-                    currentRoutePath.includes(navbar.url)
-                      ? navbar.showContactBtn
-                        ? ''
-                        : 'after:tw-w-full after:tw-scale-x-0 after:tw-scale-x-100'
-                      : '',
+                        ' after:tw-absolute after:tw-top-[27px] after:tw-bottom-0 after:tw-left-0 after:tw-w-full after:tw-h-[3px]  after:tw-bg-gradient-[90deg] after:tw-from-[#f2709c] after:tw-to-[#ff9472] after:tw-origin-bottom-left after:tw-duration-300 after:tw-scale-x-0 hover:after:tw-scale-x-100 hover:after:tw-origin-bottom-left hover:tw-bg-clip-text hover:tw-bg-gradient-[270.11deg] hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-text-transparent ',
                   ]"
                 >
                   <span
                     :class="[
-                      currentRoutePath == navbar.url && !navbar.showContactBtn
-                        ? 'v2-canopas-gradient-text'
+                      navbar.name == 'Get Free Consultation'
```

Comment:
Let's add any flag, id or constant instead of name, as its critical to handle if we want to change the name